### DISCUSSION TITLE
other engines

### DISCUSSION THREAD

**Discussion by bit-of-a-shambles** [monora/rgl#59] [Q&A] [answered]
Is it possible to use other graphviz engines, such as fdp or neato when generating graphic files?

**Comment by monora** [comment, marked as answer]
With `dot` you can specify the layout engine with the -K parameter (see https://graphviz.org/doc/info/command.html#-K). With [write_to_graphic_file](https://www.rubydoc.info/github/monora/rgl/RGL%2FGraph:write_to_graphic_file) you can create the _dot file_ and then call dot with the -K option in a second step.
